src/network/netdev/geneve.h

```diff
@@ -21,16 +21,17 @@ typedef enum GeneveDF {
 struct Geneve {
         NetDev meta;
 
+        unsigned ttl;
```

Comment:
Why do you change the type? I think `uint8_t` is better.

Reply:
I wanted to print the error in the verify but that's ok